deployment/conf_files/etc/init/vnet-vnmgr.conf

```diff
@@ -8,6 +8,10 @@ respawn limit 5 60
 env NAME=vnmgr
 
 script
+    ## log
+    #exec 2>>/var/log/openvnet/vnmgr-upstart.log
+    #set -x
```

Comment:
True. In that case can you change the first line to something like this? `## Uncomment to enable upstart debug logging.`